lib/internal/blob.js

```diff
@@ -395,13 +396,18 @@ function ClonedBlob() {
 }
 ClonedBlob.prototype[kDeserialize] = () => {};
 
+function TransferrableBlob(handle, length, type = '') {
```

Comment:
would it be faster if remove `= ''`?

Reply:
I don't think so, I don't know of any case of default parameters slowing down any function by any margin.

Reply:
yes you are right. they are identical the same

https://perf.link/#eyJpZCI6Im5vMzEzMzBrb3hzIiwidGl0bGUiOiJpbml0IHcvd28gZGVmYXVsdCBwYXJhbXMiLCJiZWZvcmUiOiJmdW5jdGlvbiBBKEYsIHAgPSAxKSB7XG4gIHJldHVybiBuZXcgRihwKVxufSIsInRlc3RzIjpbeyJuYW1lIjoiVGVzdCBDYXNlIiwiY29kZSI6ImZ1bmN0aW9uIEIocCA9IDIpIHtcbiAgdGhpcy5wID0gcDtcbn1cblxuZm9yIChsZXQgaSA9IDA7IGkgPCAxZTY7ICsraSkge1xuICBuZXcgQShCKVxufSIsInJ1bnMiOlsxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMTIsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTEyLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjFdLCJvcHMiOjEyMH0seyJuYW1lIjoiVGVzdCBDYXNlIiwiY29kZSI6ImZ1bmN0aW9uIEMocCkge1xuICB0aGlzLnAgPSBwO1xufVxuXG5mb3IgKGxldCBpID0gMDsgaSA8IDFlNjsgKytpKSB7XG4gIG5ldyBBKEMpXG59IiwicnVucyI6WzEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMTIsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMSwxMjEsMTIxLDEyMV0sIm9wcyI6MTIwfV0sInVwZGF0ZWQiOiIyMDIzLTA5LTIwVDAyOjU5OjUzLjE0OFoifQ%3D%3D